docs/infrastructure/administration.md

```diff
@@ -81,7 +81,7 @@ activity can also be seen on the "activity" tab in the Heroku dashboard for each
 [deploy-stage]: https://dashboard.heroku.com/apps/treeherder-stage/deploy/github
 [deploy-prod]: https://dashboard.heroku.com/apps/treeherder-prod/deploy/github
 [what's deployed]: https://whatsdeployed.io/s-dqv
-[travis ci build]: https://travis-ci.org/mozilla/treeherder/builds
+[circleci build]: https://app.circleci.com/pipelines/github/mozilla/treeherder
```

Comment:
the previous travis link had /builds for circleci it doesnt, is this intentional?  Also should it be s/circleci/CircleCI/

Reply:
Yeah, the pipeline shows the build history for all branches.